programs/bpf_loader/src/syscalls.rs

```diff
@@ -2118,976 +2285,3475 @@ fn call<'a>(
     Ok(SUCCESS)
 }
 
-#[cfg(test)]
-mod tests {
-    use super::*;
-    use solana_rbpf::{
-        ebpf::HOST_ALIGN, memory_region::MemoryRegion, user_error::UserError, vm::Config,
-    };
-    use solana_sdk::{
-        bpf_loader,
-        fee_calculator::FeeCalculator,
-        hash::hashv,
-        process_instruction::{MockComputeMeter, MockInvokeContext, MockLogger},
+/// BIGNUM data cost
+const BIGNUM_WORDCOST: f64 = 6.0;
```

Comment:
Stick these here: https://github.com/solana-labs/solana/blob/27b84dda4e8f7b03c8c8d15711d91842157249ff/sdk/src/process_instruction.rs#L153

Reply:
Hmmm... do you mean put the constants before the `BpfComputeBudget`? If you mean add them I could do that but I'd be adding two (2) cost values to each syscall struct if I read this correctly....

Reply:
Fixed in a156563ebf54e8934b207cb12cb2d8aefb83b93d